packages/spectrum/src/spectrum-control/InputSlider.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+  The MIT License
+
+  Copyright (c) 2020 headwire.com, Inc
+  https://github.com/headwirecom/jsonforms-react-spectrum-renderers
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+import React from 'react';
+import { CellProps } from '@jsonforms/core';
+import { merge } from 'lodash';
+import { DimensionValue } from '@react-types/shared';
+import { SpectrumInputProps } from './index';
+import { Slider } from '@react-spectrum/slider';
+
+export class InputSlider extends React.PureComponent<
+  CellProps & SpectrumInputProps
+> {
+  render() {
+    const {
+      data,
+      config,
+      enabled,
+      uischema,
+      path,
+      handleChange,
+      label,
+      visible,
+      schema,
+    } = this.props;
+
+    const appliedUiSchemaOptions = merge({}, config, uischema.options);
+
+    const onChange = (value: any) => handleChange(path, value);
+
+    const width: DimensionValue = appliedUiSchemaOptions.trim
+      ? undefined
+      : '100%';
+
+    return (
+      <Slider
+        value={data ?? ''}
+        label={label}
+        minValue={schema.minimum}
+        maxValue={schema.maximum}
+        isHidden={!visible}
+        onChange={onChange}
+        isDisabled={!enabled}
+        width={width}
+        step={schema.multipleOf || 1}
+      />
+    );
```

Comment:
Does the `trim` option exist on the slider component as well?

Reply:
hmmm. I don't know...